app/src/main/res/layout/view_synced_tabs_title.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <TextView
+        style="@style/Header16TextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="60dp"
+        android:layout_weight="1"
+        android:text="@string/synced_tabs" />
+
+    <ImageView
+        android:id="@+id/refresh_icon"
```

Comment:
nit - let's give this a `selectableItemBackground` so it visually clicks

Reply:
Same with the synced tabs items - they should have a `selectableItemBackground` 
`

Reply:
> Same with the synced tabs items - they should have a `selectableItemBackground`

I already see that the synced tab items have the ripple animation when long pressed. Are you referring to something else?

Reply:
Okay I see the ripple it's just verrrryyy faint on dark mode. Also the padding/margin is uneven when you click it - let's fix that then 😅

Reply:
Yeah! I actually noticed and fixed that because of this comment and then wasn't sure if _that_ was what you were referring to. 😁

Reply:
Brain waves 🧠 〰️